### ISSUE TITLE
Rating starts fail target size minimum requirement

### ISSUE THREAD

**Issue by MajoMaldonado** [woothemes/woocommerce#57694]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Rating Starts are only 16.8x16.8 pixels. This tap target size may be challenging for many users to interact with, particularly those with dexterity issues or limited mobility.

@vikrampm1 this is the ID WooCore-082

### Expected behavior

Increase the size of the tap target to at least 24 by 24 (44x44 if AAA) pixels as recommended by accessibility guidelines. This will make it easier for users, including those with dexterity challenges or limited mobility, to interact with the element.

### Actual behavior

Rating Starts are only 16.8x16.8 pixels. 

### Steps to reproduce

Go to https://woo.equalizedigital.com/product/intelligent-paper-knife/
Locate the rating stars, notice they fail meeting the minimum 24x24 target size.

### WordPress Environment

https://woo.equalizedigital.com/product/intelligent-paper-knife/

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.
